diagram.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/golang/freetype"
 	"github.com/golang/freetype/truetype"
 
-	"github.com/akavel/ditaa/embd"
-	"github.com/akavel/ditaa/fontmeasure"
-	"github.com/akavel/ditaa/graphical"
+	"./embd"
+	"./fontmeasure"
+	"./graphical"
```

Comment:
Sorry, I know the pain, but AFAIK relative imports are officially "Bad Go Style&trade;", so this won't go for now.

Sorry I couldn't find a better "official" reference to back the above fact fast, but this one at least hints at it: https://github.com/golang/go/issues/6147

I believe there might be a tool already somewhere on teh internets to help with this case, there must have been already someone annoyed enough...